debian/control

```diff
@@ -15,6 +15,7 @@ Build-Depends:
  python3-setuptools,
  python3-pip,
  python3-venv,
+ libdistro-info-perl,
```

Comment:
I tried building the package without this installed, and it seemed just fine. Can you explain why it is required?

Reply:
The build for buster specifically complained about needing this package, and after placing it in it built. Maybe something specific to our build machine's environment?

Reply:
given the build machine is building the packages in a docker container, I'd rather hope that environment-specific aspects of the build machine wouldn't affect the build.

> The build for buster specifically complained about needing this package, and after placing it in it built. 

Are you sure you weren't misled by the problem with sqlite? I did see this, but it's just a warning:

```
+ dch -M -r '' --force-distribution --distribution buster
libdistro-info-perl is not installed, Debian release names are not known.
libdistro-info-perl is not installed, Ubuntu release names are not known.
```

I guess we could put it in the docker build image to silence the warning, but I don't think it's needed by the actual package build process, so it shouldn't be in `Build-Depends`, and tbh I wouldn't bother with it at all.

Reply:
Got it, have removed.